### PR TITLE
Add a method to check if a Vector is a linear combination of columns of a Matrix

### PR DESCRIPTION
**Pull request by AliTavassoly**

### CODE REVIEW

**Comment by baggepinnen** [comment]
I think you could potentially avoid half of the computations by not performing the `rank` computation twice, e.g., using something like
```julia
islc2(A,y) = norm(A*(A\y) - y) < sqrt(eps())
```

The name is quite non-descriptive and I could not find any function with a similar name elsewhere. Will this function be discoverable by others?

File: stdlib/LinearAlgebra/src/generic.jl [hunk 1]
```diff
@@ -1795,3 +1795,35 @@ function normalize(a::AbstractArray, p::Real = 2)
...
+julia> y = [3, 1]
+ 2-element Vector{Int64}:
+  3
+  1
+julia> println(islc(A, y))
+  true
+```
+"""
+function islc(A::AbstractMatrix, y::AbstractVector)
+    tA = copy(A)
```
Comment (baggepinnen) [inline stdlib/LinearAlgebra/src/generic.jl:1821]: Taking the copies seem unnecessary, the arrays are not mutated?

**Comment by oscardssmith** [comment]
If we want this, (not sure we do), eps should be configurable via a keyword argument.

File: stdlib/LinearAlgebra/src/generic.jl (hunk 1 above)
Comment (AliTavassoly) [reply stdlib/LinearAlgebra/src/generic.jl:1821]: Hey @baggepinnen,
Thanks for the response and advice,
About the name, I was struggling with the naming and couldn't find a standard about how to name a method, I'd appreciate it if you could provide me with a good name for this method.
And about the copies, i'll be fixing it for the next commit,
Thanks in advance.

**Comment by dkarrasch** [comment]
I don't think this should be in the stdlib, since it answers a very specific (plus `eps`-dependent) question at a potentially very high expense.

**Comment by dkarrasch** [comment]
I'll close this one. @AliTavassoly We all appreciate your effort and your contribution, welcome to the Julia community anyways!

**Pull request closed without merging**